src/main/java/com/vecna/maven/db/mysql/MysqlStrategy.java

```diff
@@ -0,0 +1,146 @@
+/**
+ * Copyright 2014 Vecna Technologies, Inc.
```

Comment:
If this particular file is an entirely new class though, should this class be dated 2017 and the rest left 2014?